packages/rum/src/viewTracker.ts

```diff
@@ -55,8 +55,7 @@ function newView(location: Location, session: RumSession, upsertRumEvent: (event
     location: { ...location },
     sessionId: session.getId(),
   }
-  startTimestamp = new Date().getTime()
-  startOrigin = performance.now()
+  startOrigin = isInitialView ? 0 : performance.now()
```

Comment:
NIT: you could check if the `viewContext` object was defined instead of having another mutable state.